examples/event-sources/github.yaml

```diff
@@ -49,7 +49,7 @@ spec:
 #        # Name of the K8s secret that contains the hook secret
 #        name: github-access
 #        # Key within the K8s secret whose corresponding value (must be base64 encoded) is hook secret
-#        key: secret
+#        key: token
```

Comment:
Why?

Reply:
Because the `github-access` Secret only has `token` as the key. See step 3 in this document: https://argoproj.github.io/argo-events/eventsources/setup/github/

`webhookSecret` having the incorrect key leads to errors in the event source pod (something like cannot find the `argo-events/secret/github-access`).

Reply:
I see. In that case, we should change the secret to contain a `secret` key, `apiToken` is the token to access github api, `webhookSecret` is the secret used for github webhook configuration, they are different.

Reply:
Sounds good. Just updated the PR to add that instead.